dll/win32/shimgvw/shimgvw.c

```diff
@@ -921,9 +937,9 @@ ImageView_InitControls(HWND hwnd)
 
     if (shiSettings.Maximized) ShowWindow(hwnd, SW_MAXIMIZE);
 
-    hDispWnd = CreateWindowEx(WS_EX_CLIENTEDGE, WC_STATIC, _T(""),
-                              WS_CHILD | WS_VISIBLE,
-                              0, 0, 0, 0, hwnd, NULL, hInstance, NULL);
+    hDispWnd = CreateWindowExW(WS_EX_CLIENTEDGE, WC_STATIC, L"",
```

Comment:
```suggestion
    hDispWnd = CreateWindowExW(WS_EX_CLIENTEDGE, WC_STATIC, NULL,
```
Technically `NULL` for window text is valid.

Reply:
but is it an improvement?